3-prod_datascience/evaluate_model.py

```diff
@@ -10,20 +10,23 @@
     ClassificationMetrics,
 )
 
-@component(base_image="tensorflow/tensorflow", packages_to_install=["tf2onnx", "onnx", "pandas", "scikit-learn"])
+@component(base_image="tensorflow/tensorflow", packages_to_install=["tf2onnx", "onnx", "pandas", "scikit-learn", "model-registry==0.2.7a1"])
```

Comment:
```suggestion
@component(base_image="tensorflow/tensorflow", packages_to_install=["tf2onnx", "onnx", "pandas", "scikit-learn", "model-registry==0.2.9"])
```

living on the edge :p